src/renderer/render_api_opengles.cpp

```diff
@@ -2364,14 +2508,87 @@ class RHI_OpenGL : public TrRenderHardwareInterface
   }
   TR_OPENGL_FUNC void OnClear(ClearCommandBufferRequest *req, renderer::TrContentRenderer *reqContentRenderer, ApiCallOptions &options)
   {
-    if (options.executingPassType != ExecutingPassType::kOffscreenPass) [[unlikely]]
```

Comment:
This return-check-statement can't be removed, we don't allow the applications to clear the host framebuffer.